hashring/hashring_test.go

```diff
@@ -310,7 +310,8 @@ func TestLookupNLoopAround(t *testing.T) {
 	ring.AddMembers(members...)
 
 	unique := make(map[valuetype]struct{})
-	ring.tree.LookupNUniqueAt(1, replicaPoint{hash: 0}, unique)
+	orderedUnique := make([]valuetype, 0, 1)
+	ring.tree.LookupNUniqueAt(1, replicaPoint{hash: 0}, unique, &orderedUnique)
```

Comment:
If I understand this change correctly (debatable), you should be able to assert that (1) the returned slice has the correct length, (2) that the sort is stable, and (3) that the first item is the same as the returned value from `Lookup`.

@thanodl may have some ideas for more thorough testing; the only further tests I can imagine would require using the farmhash code directly.